web/packages/plugins/mermaid/src/index.tsx

```diff
@@ -76,6 +76,7 @@ function useExplorer(dmssAPI: DmssAPI) {
 
 const PluginComponent = (props: DmtUIPlugin) => {
   const { documentId, dataSourceId } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore